modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -67,8 +67,8 @@ object types {
   )
 
   object RewardsConfig {
-    val stardustPrimary: Address = Address("DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS")
-    val stardustSecondary: Address = Address("DAG8VT7bxjs1XXBAzJGYJDaeyNxuThikHeUTp9XY")
```

Comment:
I'd leave this address in place since it's not being touched / changed and not used in the same way as the others.